biohansel/utils.py

```diff
@@ -151,7 +128,6 @@ def collect_fasta_from_dir(input_directory: str) -> List[Tuple[str, str]]:
     return input_genomes
 
 
-NT_SUB = {x: y for x, y in zip('acgtrymkswhbvdnxACGTRYMKSWHBVDNX', 'tgcayrkmswdvbhnxTGCAYRKMSWDVBHNX')}
 
 
 def revcomp(s):
```

Comment:
just a small thing, but perhaps a more descriptive variable name for "s"?

Reply:
also for "c"?

Reply:
also, if the function returns a string, perhaps -> str is good?

Reply:
Changed in 47fa20fae518aee051462ab5c9116422d5bd47db